tap_activecampaign/schema.py

```diff
@@ -37,10 +37,13 @@ def get_schemas():
         mdata = metadata.to_map(mdata)
         # Loop through all keys and make replication keys of automatic inclusion
         for field_name in schema['properties'].keys():
-            
+
             if stream_metadata.get('replication_keys') and field_name in stream_metadata.get('replication_keys'):
                 mdata = metadata.write(mdata, ('properties', field_name), 'inclusion', 'automatic')
 
+            if stream_metadata.get('additional_automatic_keys') and field_name in stream_metadata.get('additional_automatic_keys'):
```

Comment:
```suggestion
            if field_name in stream_metadata.get('additional_automatic_keys', []):
```

Suggest to change the above condition